pkg/Earthfile

```diff
@@ -1,35 +0,0 @@
-VERSION 0.8
-IMPORT ../infrastructure/earthly/go AS go-build
-
-deps:
-    FROM ../+deps
-    COPY . pkg/
-    RUN cd pkg && buf generate
-    RUN cd pkg && go tool oapi-codegen -generate "std-http-server" -o publicapi/server-gen.go -package publicapi publicapi/api.yaml
-    SAVE ARTIFACT pkg/api/v1/*.go AS LOCAL api/v1/
-    SAVE ARTIFACT pkg/publicapi/* AS LOCAL publicapi/
-artifacts:
-    FROM +deps
-    SAVE ARTIFACT pkg
-lint:
-    FROM +deps
-    WORKDIR pkg
-    RUN buf lint
-    DO go-build+LINT --skip_lint_errors=false
-test:
-    FROM +deps
-    ARG GO_TEST_ARGS
-    WORKDIR pkg
-    WITH DOCKER --compose ../docker-compose-unittest.yml
-        RUN --no-cache \
-            echo "Started docker compose for unit tests." ;\
-            go test $GO_TEST_ARGS ./... -coverprofile=coverage.out
-    END
-    RUN go tool cover -html=coverage.out -o coverage.html
-    RUN ../infrastructure/coverage/check-coverage-go.sh coverage.out 47.8 pkg
```

Comment:
when you remove this, you need to add `MIN_COVERAGE=47.8` to the makefile.

Reply:
It's already being done in the `include.mk` file.